### ISSUE TITLE
Check/Uncheck task causes great damage to group Boss

### ISSUE THREAD

**Issue by mikean92** [HabitRPG/habitica#7906]
General Info
- UUID: 9ee20fbd-ed43-40c2-a900-acfc9582b902
- Group ID: bfb55d3c-2ec8-4f80-b67f-6152d2b2e32e
- Browser: Chrome 52
- OS: Linux
### Description

Checking and unchecking repeatedly a task gives and remove HP, EXP and coins. That's completely normal. But unchecking does not remove Damage deal to Boss. So if you check and uncheck a couple of times a task, you will make a huge hit unintentionally to a Boss.

**Comment by khipkin** [comment]
@mikean92 You should report this in the [Report a Bug guild](https://habitica.com/#/options/groups/guilds/a29da26b-37de-4a71-b0c6-48e72a900dac) before creating a GitHub issue just in case they can help you over there.  @Alys 's [response on a similar issue](https://github.com/HabitRPG/habitrpg/issues/6971#issuecomment-202193264) may also be helpful.

**Comment by Alys** [comment]
Thanks @khipkin !

@mikean92 We do consider this a low priority because of the way we don't try to stop cheating, as described in the issue @khipkin linked to, however we intend to fix it when we can. It's reported in https://github.com/HabitRPG/habitrpg/issues/5648 and the fix is waiting on me to do some other stuff first. :( I'm closing this but you can follow https://github.com/HabitRPG/habitrpg/issues/5648 if you want to know when it's fixed. Thank you for telling us though!

**Comment by khipkin** [comment]
@Alys Nice catch on the duplicate issue... I couldn't find one. :stuck_out_tongue_closed_eyes:
